script/spec

```diff
@@ -0,0 +1,36 @@
+#! /bin/sh
+
```

Comment:
kinda want `set -e` so it fails when 1 fails ?
also name might be better as `script/all_specs` or so
... ideally don't build a 1-off tool but reuse a GA-simulator that reads the CI yaml file
kinda like wwtd was, but I guess we can also use a simpler approach
https://github.com/zendesk/stronger_parameters/blob/master/Rakefile#L17-L25 is what I did for other repos

Reply:
Thanks. I'll investigate